src/wallet/rpcwallet.cpp

```diff
@@ -4636,6 +4724,7 @@ static const CRPCCommand commands[] =
     { "wallet",             &bumpfee,                        },
     { "wallet",             &psbtbumpfee,                    },
     { "wallet",             &createwallet,                   },
+    { "wallet",             &restorewallet,        },
```

Comment:
In commit e7c86cdb6b0f1b50816b9aec7a9ce1bfeedd32f5 "Add a new RPC command: restorewallet"

nit: whitespace, `}` should line up with the rest.

Reply:
Done.